modules/saml/docs/sp.md

```diff
@@ -259,6 +259,12 @@ Options
 
 :   *Note*: SAML 2 specific.
 
+`maxCache`
+:   maxCache allows you to set the `validUntil` in the SP's metadata. It's an integer representing the amount of seconds.
+
+`maxDuration`
+:   maxDuration allows you to set the `cacheDuration` in the SP's metadata. It's an integer representing the amount of seconds.
```

Comment:
I'm not so happy with the naming.

`maxCache` sets validUntil while the SAML MD spec description of `cacheDuration` is "maximum length of time a consumer should cache". So I suspect that people will be confused thinking that `maxCache` is related to the cache time of the metadata.

Probably it's most clear if the options are named according to the corresponding metadata parameters they set. So "maxCache" could be named "validUntil" and the help text coild explain that after this time an entity is expired and unusable. "maxDuration" could be named "cacheDuration" and the help text can say that this is the suggested time a remote entity should use between refresh attempts.

Reply:
It's actually the other way around.. My added docs are wrong :/   I'll rename them and fix the docs